cli/internal/fs/turbo_json.go

```diff
@@ -92,13 +134,26 @@ type TaskDefinition struct {
 	Persistent bool
 }
 
+// ResolvedTaskDefinition is a modified version the TaskDefinition struct.
+// It is meant for merged task definitions and for printing out in Run Summaries.
+type ResolvedTaskDefinition struct {
+	Outputs                 *TaskOutputs
```

Comment:
Looks like I can remove `ResolvedTaskDefinition` entirely. I think the requirement to a pointer to `TaskOutputs` may have been something I introduced early along the journey and didn't question it after that. Since the tests are passing without that requirement, this whole struct can go away.